src/lib/shell/commands/Ota.cpp

```diff
@@ -167,7 +169,16 @@ void OnQueryImageFailure(void * /* context */, EmberAfStatus status)
     ChipLogError(SoftwareUpdate, "QueryImage failed: %" PRIu16, static_cast<uint16_t>(status));
 }
 
-void OnQueryImageConnection(void * /* context */, DeviceProxy * deviceProxy)
+} // namespace
+} // namespace Shell
+} // namespace chip
+
+#include <controller-clusters/zap-generated/CHIPClusters.h>
```

Comment:
Why has it been moved here?

Reply:
I moved the include line here to avoid an error `Include not allowed` when compiling all-clusters-app.